packages/ssi-types/src/types/vc.ts

```diff
@@ -195,6 +195,8 @@ export interface JwtDecodedVerifiablePresentation {
 }
 
 export type ClaimFormat = 'jwt' | 'jwt_vc' | 'jwt_vp' | 'ldp' | 'ldp_vc' | 'ldp_vp' | string
```

Comment:
```suggestion
export type ClaimFormat = CredentialFormat | ClaimFormat
```

Reply:
Needs to be moved below the other properties probably

Reply:
moved claim lower than `CredentialFormat` and `PresentationFormat`